value/src/main/java/com/google/auto/value/processor/AutoValueProcessor.java

```diff
@@ -494,7 +504,7 @@ private void fixReservedIdentifiers(Map<ExecutableElement, String> methodToIdent
   }
 
   private String disambiguate(String name, Collection<String> existingNames) {
-    for (int i = 0; ; i++) {
```

Comment:
This change has nothing to do with the commit. But I fixed it because checkstyle was complaining about it.